test/functional/system/common.js

```diff
@@ -135,6 +135,10 @@ function forge(library, cb) {
 	);
 }
 
+function popLastBlock(library, cb) {
```

Comment:
Rename to `deleteLastBlock`, as we already have `popLastBlock` function that is private and called by `deleteLastBlock` ;)